PyPDF2/generic.py

```diff
@@ -82,15 +82,14 @@ def readObject(stream, pdf):
         # comment
         while tok not in (b_('\r'), b_('\n')):
             tok = stream.read(1)
-            # Prevents an infinite loop by raising an error if the stream is at
```

Comment:
It looks not very good to remove something that prevents infinite loops ... is that no longer necessary?